client/components/nav.jsx

```diff
@@ -4,6 +4,8 @@ import Nav from 'react-bootstrap/lib/Nav';
 import NavItem from 'react-bootstrap/lib/NavItem';
```

Comment:
React-bootstrap modules should be in one line to maintain consistency and improve brevity